tests/run_on_chain.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 rm -f tx.json tx.yaml
 
+if [ -z "${NODE_URL}" ]; then
+  CHAINWEB_NODE='https://api.chainweb.com'
+else
+  CHAINWEB_NODE=${NODE_URL}
+fi
+
 kda gen -t on_chain.tkpl
 kda combine-sigs tx.yaml
-STATUS=`kda local tx.json|jq -r ".[]|.[].body.result.status"`
+STATUS=`kda local tx.json -n ${CHAINWEB_NODE}|jq -r ".[]|.[].body.result.status"`
```

Comment:
I wasn't able to use the `kda` commands in this file without specifying a node, but if you prefer a different solution to what I've done here I can update accordingly.